### ISSUE TITLE
Support AARCH64

### ISSUE THREAD

**Issue by namero999** [criteo/JVips#110]
Hello, I was wondering if support for aarch64 (for both build process and prepackaged jar) is planned.

I am currently building on aarch64 and making progress each time I encounter a problem, but couldn't yet complete the build... will report once I manage to.

**Comment by namero999** [comment]
Had a successful build. I attach it here in case someone is looking for it (the file is actually a plain jar, adding zip extension to make github happy).

[JVips-aarch64.jar.zip](https://github.com/criteo/JVips/files/7919462/JVips-aarch64.jar.zip)

**Comment by dbouron** [comment]
Hello,

No we didn't plan a build on this arch, can you please share your changes to get a successful build?

**Comment by namero999** [comment]
I have edited `Toolchain-linux.cmake` and changed `set(CMAKE_SYSTEM_PROCESSOR AMD64)` to `set(CMAKE_SYSTEM_PROCESSOR AARCH64)`.

Then, the rest was mostly fighting with dependencies since I was building on an Oracle Linux 8.

The main issue has been dealing with `gtkdocize` which doesn't seem to be included in the repo, and I had to retrofit [this rpm](https://centos.pkgs.org/7/centos-aarch64/gtk-doc-1.28-2.el7.aarch64.rpm.html) for centos (installed it with `--nodeps` otherwise it would complain about other dependencies).

Few of the packages listed on the [docker file](https://github.com/criteo/JVips/blob/master/.github/docker/linux/Dockerfile) were also not existent on Orale Linux but they don't seem to prevent a successful build. Ultimately it has been half a day of dependency hell but eventually the build ran smooth.

All of this to ultimately be unable to use the lib, since rotation is not yet supported :) I'd like to contribute but not sure how (not super comfortable with native bindings).
